config/plugin/windows_defender_problem.ps1

```diff
@@ -0,0 +1,12 @@
+# This plugin checks to see if windows defender detects any threats to the node.
+
+$OK=0
+$NONOK=1
+
+$windowsDefenderThreats = Get-MpThreatDetection
+
+if ($windowsDefenderThreats.length -ne 0) {
```

Comment:
(optional) If it's tolerable from the custom plugin logic. I'd print a list of the threats that were found.

Reply:
yea, I think that's a good idea.